app/helpers/application_helper.rb

```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = "Ruby on Rails Tutorial Sample App"
```

Comment:
1. tìm hiểu về `I18n in rails ` nhé

2. Phân biệt cho chị nghe: blank? và empty? khác nhau thế nào

Reply:
blank? : 
- Là hàm trong rails
- Sử dụng được đối với bất kỳ đối tượng nào
- Trả về true nếu đối tượng có giá trị là nil/ false/ empty , ngược lại false

empty? : 
- Là hàm của ruby
- Không sử dụng được đối với any object, thường được sử dụng với array, hash, string
- Trả về true nếu đối tượng có giá trị là empty, ngược lại false